pkg/webhook/resources/setting/validator.go

```diff
@@ -1304,8 +1308,31 @@ func (v *settingValidator) checkStorageNetworkRangeValid(config *storagenetworkc
 		return err
 	}
 	if !network.IP.Equal(ip) {
-		return fmt.Errorf("Range should be subnet CIDR %v", network)
+		return fmt.Errorf("range should be subnet CIDR %v", network)
 	}
+
+	lhnodes, err := v.lhNodeCache.List(metav1.NamespaceAll, labels.Everything())
+	if err != nil {
+		return werror.NewInternalError(err.Error())
+	}
+
+	MinallocatableIPAddrs := 0
+	nodeNum := 1
+	//Required Number of IPs = (Number of Nodes * 2) + (Number of Nodes * Number of Disks * 2) + Number of Images to Download/Upload
+	//Number of Images to download/upload is dynamic, so skipped in the formula calculated.
+	for _, lhNode := range lhnodes {
+		MinallocatableIPAddrs = MinallocatableIPAddrs + ((nodeNum * 2) + (nodeNum * len(lhNode.Spec.Disks) * 2))
+	}
+
```

Comment:
```suggestion
	//Required Number of IPs = (Number of Nodes * 2) + (Number of Nodes * Number of Disks * 2) + Number of Images to Download/Upload
	//Number of Images to download/upload is dynamic, so skipped in the formula calculated.
	for _, lhNode := range lhnodes {
		MinAllocatableIPAddrs = MinAllocatableIPAddrs + ( 2 + len(lhNode.Spec.Disks) * 2)
	}
```
IIUC, the variable `nodeNum` should not be necessary?

Reply:
yes, you are right, I have corrected and updated the review